include/drivers/sensor.h

```diff
@@ -277,6 +280,18 @@ enum sensor_attribute {
 	 * offset: final_value = sensor_value + offset.
 	 */
 	SENSOR_ATTR_OFFSET,
+
+	/**
+	 * The sensor value returned will be altered by the amount indicated by
+	 * slope: final_value = sensor_value * slope.
+	 */
+	SENSOR_ATTR_SLOPE,
+
+	/**
+	 * The sensor gain.
+	 */
+	SENSOR_ATTR_GAIN,
```

Comment:
These can now be removed

Reply:
I do not understand why they need to be removed.
How are those attributes supposed to be changed during runtime?

**sensor_attr_set** still requires an **enum sensor_attribute attr**